src/event_pusher/mod_event_pusher_push.erl

```diff
@@ -74,26 +74,31 @@
 %%--------------------------------------------------------------------
 %% gen_mod callbacks
 %%--------------------------------------------------------------------
--spec start(Host :: jid:server(), Opts :: list()) -> any().
+-spec start(HostType :: mongooseim:host_type(), Opts :: gen_mod:module_opts()) -> any().
```

Comment:
Minor, I think that variable names should agree between the spec and the implementation. We could either change to `HostType` or leave `Host` in both places.